lib/public-api.js

```diff
@@ -9,3 +10,5 @@ exports.setThePassword = require("./url-state-machine").setThePassword;
 exports.serializeHost = require("./url-state-machine").serializeHost;
 exports.serializeInteger = require("./url-state-machine").serializeInteger;
 exports.parseURL = require("./url-state-machine").parseURL;
+exports.serializeUrlencoded = require("./url-state-machine").serializeUrlencoded;
+exports.parseUrlencodedString = require("./url-state-machine").parseUrlencodedString;
```

Comment:
Let's just make its name `parseUrlencoded` here and throughout; no need to state the argument type.